train.py

```diff
@@ -492,7 +492,7 @@ def eval_model(global_step, writer, device, model, y, c, g, input_lengths, eval_
     y_target = y[idx].view(-1).data.cpu().numpy()[:length]
 
     if c is not None:
-        c = c[idx, :, :length].unsqueeze(0)
+        c = c[idx, :, :int(length / audio.get_hop_size())].unsqueeze(0)
```

Comment:
If `upsample_conditional_features` is False (which was default before), `c` has same time length as `y`. So what about the following?

```py
if hparams.upsample_conditional_features:
  c[idx, :, :length // audio.get_hop_size()].unsqueeze(0)
else:
  c = c[idx, :, :length].unsqueeze(0)
```

Reply:
Yeah that's fine.